Started with a description of studies in molecular mechanism section (#347)
greenelab/covid19-review: content/07.pathogenesis.md

They analyzed the responses of three human cell lines: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
As the viral receptor ACE2 has low expression in A549 cells, authors also supplemented A549 cells with a vector expressing mCherry or ACE2 (A549-ACE2). 
All infections were performed at a high multiplicity of infections (MOI, 2–5), except A549-ACE2 cells were infected with SARS-CoV-2 at both low (0.2 MOI) and high MOI (2 MOI).
Poly(A) bulk RNA sequencing (RNA-seq) of host cells was performed after 24 hours post-infection (hpi) of all the viruses except IAV (9-12 hours).
The authors also measured host transcriptional responses to SARS-CoV-2 in primary normal human bronchial epithelial cells (HBEC or NHBE cells), nasal washes of ferrets as an animal model, and lung samples from two COVID-19 patients. 
Additionally, transcriptional response to IAV in NHBE cells and nasal washes of ferrets was measured.
Differential expression (DE) analysis was carried out between infected and respective mock (control) cell condition.

> DE results revealed that the transcriptional response of the cells with efficient SARS-CoV-2 replication is significantly different from the host response of all other viruses tested. 

Reviewer: Is that signifies pathogenesis of SARS-CoV-2 and host responses to this pathogen are substantially different from other respiratory viruses? A brief analysis/discussion on this imperative observation made by the authors of this manuscript will be interesting and informative for the readers.

Author: Thanks. Yes, I discuss this in next lines (cytokines and IFNs).

A unique proinflammatory cytokine signature is present in both high and low multiplicity of infections (MOI). 

> However, in contrast to high-MOI, low-MOI SARS-CoV-2-infected A549-ACE2 cells showed no significant IFN-I or IFN-III expression, suggesting that IFN induction is MOI dependent. 

Reviewer 2: ```suggestion
However, A549-ACE2 cells in the low-MOI condition showed no significant IFN-I or IFN-III expression, but cells at high-MOI did.
This finding suggested that IFN induction is MOI dependent. 
```
I tried to space it out a little to make it easier to follow, since there are lots of acronyms!
Could you possibly add a sentence saying why we care about IFN expression levels? Are these two IFN in particular important?

However, the low IFN induction in ACE2-expressing A549 cells could be overcome by using a 10-fold increase in SARS-CoV-2 (MOI, 2), suggesting a SARS-CoV-2 antagonist that is rendered ineffective under high-MOI conditions [@doi:10.1016/j.cell.2020.04.026]. 
This hypothesis was further supported by a recent study [@doi:10.1101/2020.05.11.088179] that showed that the SARS-CoV-2 _ORF3b_ gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV-1 _ORF3b_ gene, which is considerably longer.

A second study by Wyler et al. [@doi:10.1101/2020.05.05.079194] examined three human cell lines: H1299 (human non-small cell lung carcinoma cell line), Calu-3, and Caco-2 (human epithelial colorectal adenocarcinoma cell line). 
They performed a comprehensive analysis of the transcriptional response to SARS-CoV-2 and SARS-CoV-1 at different hpi (4-36 hours) with an MOI of 0.3. 
Using poly(A) bulk RNA-seq, the authors found negligible susceptibility of H1299 cells (< 0.08 viral read percentage of total reads) in contrast to Caco-2 and Calu-3 cells (>10% of viral reads). 

> Based on the visual inspection (microscopy images) and transcriptional profiling, authors showed distinct host responses after the infection. 

Reviewer 2: ```suggestion
Based on visual inspection (microscopy images) and transcriptional profiling, the authors showed distinct host responses after the infection. 
```

> In contrast to Caco-2, SARS-CoV-2 infected Calu-3 cells showed signs of impaired growth and cell death at 24hpi, and a moderate IFN induction with a strong up-regulation of ISGs. 

Reviewer 2: ```suggestion
In contrast to Caco-2, SARS-CoV-2 infected Calu-3 cells showed signs of impaired growth and cell death at 24 hpi, as well as moderate IFN induction with a strong up-regulation of ISGs. 
```

> Interestingly, the response is similar to SARS-CoV-2 infected Calu-3 cells at ~7-fold higher MOI performed by Blanco-Melo et al., suggesting that IFN induction is not MOI dependent in Calu-3 cells (in contrast to A549-ACE2 cells). 

Reviewer 2: ```suggestion
Interestingly, the response is similar to SARS-CoV-2 infected Calu-3 cells at ~7-fold higher MOI performed by Blanco-Melo et al., suggesting that IFN induction is not MOI dependent in Calu-3 cells, in contrast to reports using A549-ACE2 cells.
```

The discrepancy could be explained by the observations that Calu-3 cells are highly susceptible to SARS-CoV-2 with rapid virus replication [@doi:10.1038/s41467-020-15562-9], whereas A549 cells are incompatible with SARS-CoV-2 infection [@doi:10.1101/2020.03.02.972935]. 

> We performed a correlation analysis of the host expression response (RNA fold change compared to the mock infection) using the publicly available result files from Blanco-Melo et al. and Wyler et al. (Figure {@fig:cell-lines-moi}B) [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194]. 

Reviewer: "result files from"  please replace the phrase as "publicly available data from"

> In general, we found high correlations among the same host (e.g. Calu-3 cells) irrespective of the study and the virus infected, suggesting that the response is host-specific. 

Reviewer: Please define "high correlations" in terms of a statistical measurement (such as Pearson correlation coefficient > XX)

> Moreover, COVID-19 patients have low correaltions with all the cells used (highest value of 0.29), suggesting that the models used may not necessarily imitate the human response, therefore, appealing/asking/demanding for using additional models. 

Reviewer 2: ```suggestion
Moreover, COVID-19 patients had the lowest correlations among all the cells used, with the highest value being 0.29.
This discrepancy suggests that the _in vitro_ models used may not necessarily imitate the human response, underscoring the importance of follow-up in additional models. 
```

Reviewer: This happens very often when the observations obtained using the cell line models are compared with in vivo responses. It is a bit expected to observe such differences in host responses in cell line models of infection vs. in the patients suffering from the infection, precisely due to the complex and dynamic physiological and metabolic systems in our body.

> 

Reviewer 2: ```suggestion
```

Although we make an effort to evaluate all relevant studies, we realize that this area is rapidly evolving.

Two studies profiled expression response after SARS-CoV-2 infection in multiple human cell lines (Figure {@fig:cell-lines-moi}A) [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194]. 
The first study, which is by Blanco-Melo et al., performed an in-depth analysis of the transcriptional response to SARS-CoV-2 in comparison to other respiratory viruses, including MERS-CoV, SARS-CoV-1, human parainfluenza virus 3 (HPIV3), respiratory syncytial virus (RSV), and influenza A virus (IAV). 
They analyzed the responses of three human cell lines: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
As the viral receptor ACE2 has low expression in A549 cells, authors also supplemented A549 cells with adenovirus (AdV)-based vectors expressing mCherry (control) or ACE2 (A549-ACE2). 

> All infections were performed at a high multiplicity of infections (MOI, 2–5), with the exception of A549-ACE2 cells, which were infected with SARS-CoV-2 at both low (0.2 MOI) and high MOI (2 MOI).

Reviewer 2: @bansalvi just an FYI, in the future if you want to incorporate changes made with the suggestion interface (like above), you can just click "commit suggestion" (if you're just doing one) or "add suggestion to batch" (if you're doing a bunch) and they will automatically be changed, kind of like Track Changes in Microsoft Word!

Poly(A) bulk RNA sequencing (RNA-seq) of host cells was performed at 24 hours post-infection (hpi) of all the viruses except IAV (9-12 hours).
The authors also measured host transcriptional responses to SARS-CoV-2 in primary normal human bronchial epithelial cells (HBEC or NHBE cells), nasal washes of ferrets as an animal model, and lung samples from two COVID-19 patients. 
Additionally, transcriptional response to IAV in NHBE cells and nasal washes of ferrets was measured.
Differential expression (DE) analysis was carried out between infected and respective mock (control) cell condition.

> DE analysis revealed that the transcriptional response of the cells to efficient SARS-CoV-2 replication was significantly different from the host response to all of the other viruses tested. 

Reviewer 2: ```suggestion
In the hosts where SARS-CoV-2 was able to replicate efficiently, DE analysis revealed that the transcriptional response was significantly different from the response to all of the other viruses tested. 
```

A unique proinflammatory cytokine signature associated with SARS-CoV-2 was present under both high and low MOI conditions. 

> However, A549-ACE2 cells in the low-MOI condition showed no significant IFN-I or IFN-III expression, but cells at high-MOI did.

Author: @rando2 This is a very good question. I read it again and realised it definitely needs 1-2 general lines. Will fix this.

This finding suggested that IFN induction is MOI dependent. 
Similarly, in cells from the NHBE line, ferrets, and COVID-19 patients, they found significant enrichment of chemokine signaling but no significant induction of IFN-I or IFN-III. 
Together, these results suggest that in contrast to common respiratory viruses, SARS-CoV-2 induces limited antiviral state with low IFN-I or IFN-III expression and a moderate IFN-stimulated genes (ISG) response. 
However, the low IFN induction in ACE2-expressing A549 cells could be overcome by using a 10-fold increase in SARS-CoV-2 (MOI, 2), suggesting a SARS-CoV-2 antagonist that is rendered ineffective under high-MOI conditions [@doi:10.1016/j.cell.2020.04.026]. 

> This hypothesis was further supported by a recent study [@doi:10.1101/2020.05.11.088179] that showed that the SARS-CoV-2 _ORF3b_ gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV-1 _ORF3b_ gene, which is considerably longer.

Reviewer 2: ```suggestion
This hypothesis was further supported by a recent study [@doi:10.1101/2020.05.11.088179] that showed that the SARS-CoV-2 _ORF3b_ gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV_ORF3b_ gene, which is considerably longer.
```

A second study by Wyler et al. [@doi:10.1101/2020.05.05.079194] examined three human cell lines: H1299 (human non-small cell lung carcinoma cell line), Calu-3, and Caco-2 (human epithelial colorectal adenocarcinoma cell line). 
They performed a comprehensive analysis of the transcriptional response to SARS-CoV-2 and SARS-CoV-1 at different hpi (4-36 hours) with an MOI of 0.3. 

> Using poly(A) bulk RNA-seq, the authors found negligible susceptibility of H1299 cells (< 0.08 viral read percentage of total reads) in contrast to Caco-2 and Calu-3 cells (>10% of viral reads). 

Reviewer 2: What is different about these cell lines? What does it tell us that some are more susceptible?

Author: suggesting a role of cell-type specific host factors.

> Based on visual inspection (microscopy images) and transcriptional profiling, the authors showed distinct host responses after the infection. 

Reviewer 2: ```suggestion
Based on visual inspection (microscopy images) and transcriptional profiling, the authors showed distinct responses associated with different hosts following infection. 
```
Suggestion to incorporate the response to @rays1987's question

In contrast to Caco-2, SARS-CoV-2 infected Calu-3 cells showed signs of impaired growth and cell death at 24 hpi, as well as moderate IFN induction with a strong up-regulation of ISGs. 
Interestingly, the response is similar to SARS-CoV-2 infected Calu-3 cells at ~7-fold higher MOI performed by Blanco-Melo et al., suggesting that IFN induction is not MOI dependent in Calu-3 cells, in contrast to reports using A549-ACE2 cells.
The discrepancy could be explained by the observations that Calu-3 cells are highly susceptible to SARS-CoV-2 with rapid virus replication [@doi:10.1038/s41467-020-15562-9], whereas A549 cells are incompatible with SARS-CoV-2 infection [@doi:10.1101/2020.03.02.972935]. 
We performed a correlation analysis of the host expression response (RNA fold change compared to the mock infection) using the publicly available analyzed data from Blanco-Melo et al. and Wyler et al. (Figure {@fig:cell-lines-moi}B) [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194]. 
In general, we found high correlations among the same host irrespective of the study and the virus infected, suggesting that the response is host-specific. 

> For instance, Calu-3 cells infected with SARS-CoV-1 by Wyler et al. and Calu-3 cells infected with SARS-CoV-2 by Blanco-Melo et al. had a Pearson's correlation coefficient of 0.8 (Figure {@fig:cell-lines-moi}B).

Reviewer 2: I think we need to give the degrees of freedom and p-value, per <https://www.socscistatistics.com/tutorials/correlation/default.aspx>

Author: @rando2 I agree that it is recommended to report at least p-value. But here, I am talking about relatively high/low correlations. If you think its necessary, I will calculate p-values for all the correlations reported in Figure part B?

> For instance, Calu-3 cells infected with SARS-CoV-1 by Wyler et al. and Calu-3 cells infected with SARS-CoV-2 by Blanco-Melo et al. had a Pearson's correlation coefficient of 0.8 (Figure {@fig:cell-lines-moi}B).

Reviewer 2: ```suggestion
For instance, Calu-3 cells infected with SARS-CoV by Wyler et al. and Calu-3 cells infected with SARS-CoV-2 by Blanco-Melo et al. had a Pearson's correlation coefficient of 0.8 (Figure {@fig:cell-lines-moi}B).
```

Interestingly, the response is similar to SARS-CoV-2 infected Calu-3 cells at ~7-fold higher MOI performed by Blanco-Melo et al., suggesting that IFN induction is not MOI dependent in Calu-3 cells, in contrast to reports using A549-ACE2 cells.
The discrepancy could be explained by the observations that Calu-3 cells are highly susceptible to SARS-CoV-2 with rapid virus replication [@doi:10.1038/s41467-020-15562-9], whereas A549 cells are incompatible with SARS-CoV-2 infection [@doi:10.1101/2020.03.02.972935]. 
We performed a correlation analysis of the host expression response (RNA fold change compared to the mock infection) using the publicly available analyzed data from Blanco-Melo et al. and Wyler et al. (Figure {@fig:cell-lines-moi}B) [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194]. 
In general, we found high correlations among the same host irrespective of the study and the virus infected, suggesting that the response is host-specific. 
For instance, Calu-3 cells infected with SARS-CoV-1 by Wyler et al. and Calu-3 cells infected with SARS-CoV-2 by Blanco-Melo et al. had a Pearson's correlation coefficient of 0.8 (Figure {@fig:cell-lines-moi}B).

> Moreover, COVID-19 patients had the lowest correlations among all the cells used, with the highest value being 0.29.

Reviewer 2: See above about regression reporting

Reviewer 2: @bansalvi can you also post your code or Excel file or however you did this analysis? I can double-check the formatting guidelines for _mSystems_ and make sure the analysis is reported correctly! We will probably also want to expand the methods to cover how this was done.

Author: @rando2  @agitter  https://github.com/bansalvi/COVID19-Greene-Cor/blob/master/Cor_TenOverLandthaler.R

This discrepancy suggests that the _in vitro_ models used may not necessarily imitate the human response, underscoring the importance of follow-up in additional models. 

![**Profiling and correlation of expression response after respiratory viruses infection in various hosts.** (A) Overview of hosts and respiratory viruses used in two studies [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194]. MOI:Multiplicity of infections; PFU:Plaque-forming unit; a:ACE2-expressing A549 cells; b:Data used from GSE56192. (B) Pearson's correlation of expression response (Log2 fold change) between various hosts. Note that we used the data from bulk RNA-seq produced after the infection of unmodified viruses.](images/cell-lines-moi.jpg){#fig:cell-lines-moi}

> [Start with scRNA-seq studies in healthy tissues (may be put this before Blanco-Melo et al), then summarize scRNA-seq studies in COVID-19 patients]

Reviewer 2: ```suggestion
<!-- Start with scRNA-seq studies in healthy tissues (may be put this before Blanco-Melo et al), then summarize scRNA-seq studies in COVID-19 patients] -->
```
